packages/@aws-cdk/aws-appsync/test/appsync-opensearch.test.ts

```diff
@@ -15,7 +15,7 @@ beforeEach(() => {
     schema: appsync.Schema.fromAsset(path.join(__dirname, 'appsync.test.graphql')),
   });
   domain = new opensearch.Domain(stack, 'OsDomain', {
-    version: opensearch.EngineVersion.OPENSEARCH_1_1,
+    version: opensearch.EngineVersion.OPENSEARCH_1_2,
```

Comment:
no need to change this -- we should still test that `OPENSEARCH_1_1` works because that is a valid version